build-source-tarball.sh

```diff
@@ -124,6 +176,11 @@ for srcDir in `find bin/src -name '.git' -type d`; do
 done
 popd
 
+# now we don't need bin/src at all
+if [ $MINIMIZE_DISK_USAGE -eq 1 ]; then
+    rm -rf "$SCRIPT_ROOT/bin/src/"
```

Comment:
I think this deletes all production repo binlogs before they get copied in the next step.

Reply:
Nice catch, thanks.  Removed this part.